pom.xml

```diff
@@ -1130,7 +1130,7 @@
                 <enabled>false</enabled>
             </snapshots>
             <id>geowave</id>
-            <url>http://geowave-maven.s3-website-us-east-1.amazonaws.com/release</url>
+            <url>https://geowave-maven.s3-website-us-east-1.amazonaws.com/release</url>
```

Comment:
Have you tested this (by clearing the associated artifacts out of your maven repo and trying the build with this change)? When I last looked at changing this to https, I found that the repo only listens on http, so it wouldn't be possible to fix. However, the repo owner may have updated since then.

Reply:
Still looks like this doesn't support HTTPS. 😢 Feel free to close this.
Alternatively, you can fix this by vendoring the JAR's inside of your repository or contacting the maintainers to let them know about this security vulnerability.

Reply:
@jwomeara, can we switch to a newer geowave? I thought their newer stuff is going into maven central.

Reply:
Created #816 to solve this issue.

Reply:
Latest PR (#819) has us pulling these dependencies from Maven central.